osu.Game/Screens/IOsuScreen.cs

```diff
@@ -39,9 +39,9 @@ public interface IOsuScreen : IScreen
         bool HideOverlaysOnEnter { get; }
 
         /// <summary>
-        /// Whether overlays should be able to be opened once this screen is entered or resumed.
+        /// Whether overlays should be able to be opened when this screen is current.
         /// </summary>
-        OverlayActivation InitialOverlayActivationMode { get; }
+        Bindable<OverlayActivation> OverlayActivationMode { get; }
```

Comment:
Should definitely be immutable from the interface, value changes should only be done in the screen class itself internally.